src/libs/elektra/keyset.c

```diff
@@ -559,31 +574,27 @@ static int keyCompareByNameOwner (const void * p1, const void * p2)
 }
 
 /**
- * Compare the name of two keys.
- *
- * @return a number less than, equal to or greater than zero if
- *    k1 is found, respectively, to be less than, to match, or
- *    be greater than k2.
+ * Compare the name of two Keys.
  *
- * The comparison is based on a strcmp of the keynames, and iff
- * they match a strcmp of the owner will be used to distuingish.
- * If even this matches the keys are found to be exactly the
+ * The comparison is based on a strcmp of the Key's names, and iff
+ * they match, a strcmp of the owner will be used to distuingish.
+ * If this also matches, the Keys are found to be exactly the
```

Comment:
`keyCompareByNameOwner`/`keyCompareByOwner` is now obsolete and very soon removed with a PR.
Also `memcmp` is used.

```suggestion
 * The comparison is based on a memcmp of the Key's names.
 * If the names match, the Keys are found to be exactly the
```